src/test/scala/com/datastax/gatling/plugin/request/CqlRequestActionSpec.scala

```diff
@@ -29,12 +33,19 @@ class CqlRequestActionSpec extends BaseSpec with TestKitBase {
   implicit lazy val system:ActorSystem = ActorSystem()
   val gatlingTestConfig: GatlingConfiguration = GatlingConfiguration.loadForTest()
   val dseSession: DseSession = mock[DseSession]
-  val dseCqlStatement: DseCqlStatement = mock[DseCqlStatement]
-  val pagingState: PagingState = mock[PagingState]
+  val dseCqlStatement: DseCqlStatement[SimpleS,SimpleB] = mock[DseCqlStatement[SimpleS,SimpleB]]
+  val node:Node = mock[Node]
+  val pageSize = 3
+  val pagingState: ByteBuffer = mock[ByteBuffer]
+  val queryTimestamp = 123L
+  val routingKey:ByteBuffer = mock[ByteBuffer]
+  val routingKeyspace = "some_keyspace"
+  val routingToken:Token = mock[Token]
+  val timeout:Duration = Duration.ofHours(1)
```

Comment:
Duration is a final class so just instantiate what you need here; trying to make a mock out of this makes the JVM very angry.